app/frontend/src/views/Submission.vue

```diff
@@ -1,525 +1,536 @@
 <template>
   <v-container>
     <BaseSecure admin>
-      <router-link :to="{ name: 'Admin'}">
-        <v-btn color="primary" class="mt-5">Back</v-btn>
-      </router-link>
+      <v-alert v-if="error" type="error" tile dense>An erorr occurred fetching this submission.</v-alert>
+      <div v-else>
+        <router-link :to="{ name: 'Admin'}">
```

Comment:
Missing space between ' and }. Run a file format.

Reply:
got it